tests/phpunit/Compat/WC_Stripe_Subscriptions_Helper_Test.php

```diff
@@ -21,4 +23,124 @@ class WC_Stripe_Subscriptions_Helper_Test extends WP_UnitTestCase {
 	public function test_is_subscriptions_enabled() {
 		$this->assertTrue( WC_Stripe_Subscriptions_Helper::is_subscriptions_enabled() );
 	}
+
+	/**
+	 * Test for `get_detached_subscriptions`.
+	 *
+	 * @return void
+	 */
+	public function test_get_detached_subscriptions() {
+		add_filter(
+			'wc_stripe_unit_test_get_subscription_time_next_payment_date',
+			function () {
+				return strtotime( '+1 week' );
+			}
+		);
```

Comment:
Style-wise, might it be clearer to use a local variable so it's super-explicit that we're adding and removing the same function to the filter?
```suggestion
		$return_now_plus_one_week = function () {
			return strtotime( '+1 week' );
		};
		add_filter(
			'wc_stripe_unit_test_get_subscription_time_next_payment_date',
			$return_now_plus_one_week
		);
```